pkg/lib/config/configsource/kubernetes.go

```diff
@@ -190,19 +191,39 @@ func (k *Kubernetes) Close() error {
 }
 
 func (k *Kubernetes) AllAppIDs() ([]string, error) {
-	appIDs := k.appIDs.Load().([]string)
+	// FIXME(k8s): remove this after introducing proper authz
+	ingresses, err := k.Client.NetworkingV1beta1().Ingresses(k.Namespace).List(metav1.ListOptions{})
```

Comment:
I guess we can pass a label selector to `List` to make the list result more relevant?

Reply:
This is just a temporary fix for the lack of proper authz, and should be removed after a proper authz mechanism is in place, so I don't think it need to be very performant.

Reply:
Fair enough.